test/unit/spec/EnrollTotpController_spec.js

```diff
@@ -927,6 +936,7 @@ function (_, $, Q, OktaAuth, LoginUtil, StringUtil, Util, DeviceTypeForm, Barcod
             test.passCodeForm.backLink().click();
             return Expect.waitForManualSetupPush(test);
           })
+          .then(tick)
```

Comment:
what is this ```tick``` for? If we are waiting for the list of factors to enroll, it might be better to use a wait for a specific controller instead. I think we have already that wait: ```waitForEnrollChoices```

Reply:
Replaced with the waitFor...